### PR TITLE
WM_3-Donna-Henry-Git/GitHub-Week_2

### PR DESCRIPTION
**Pull request by dotdashdon**
This is my first attempt at correcting the homework. Thank you for looking. 

# Your Details

- Your Name: Donna Henry
- Your City: Birmingham/West Midlands 
- Your Slack Name: @Donna 
Slack ID U025Z8GMN3D

# Homework Details

- Module: Git and Git Hub
- Week: 2 of FullStack

-----
[View rendered homework.md](https://github.com/dotdashdon/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-You delete it
+You copy it and create another version but on your own profile/github website
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You copy it and create another version but on your own local computer
```
Comment (Halla-S) [inline homework.md:67]: After that you can make changes on it without affecting the origin .

File: homework.md [hunk 2]
```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+Yes, I expect that is true -When people want to show off code to each other they can put it on Github
+
+Also....To store code, to illustrate the changes and history, to share code and work together.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it and create another version but on your own profile/github website
```
Comment (Halla-S) [inline homework.md:62]: After that you can make changes on it without affecting the origin .

Comment (dotdashdon) [reply homework.md:62]: Thank you. Good point!

File: homework.md [hunk 3]
```diff
@@ -4,31 +4,31 @@
...
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Styling and layout of a website 
 
 ## Question 6
```
Comment (Halla-S) [inline homework.md:34]: You might have missed to correct this one

**Review by Halla-S — CHANGES_REQUESTED** [review]

**Review by Halla-S — COMMENTED** [review]
I have finished my review .

**Pull request closed without merging**